SilentPatch/Common_ddraw.cpp

```diff
@@ -229,6 +231,14 @@ namespace Common {
 			}
 			TXN_CATCH();
 
+			// III: Patch the icon handle to fix missing window icon
+			// (This is fixed since VC)
+			try
+			{
+				auto addr = get_pattern("c7 44 24 1c 00 00 00 00 c7 44 24 08 00 20 00", 0x4);
+				Patch<HICON>(addr, wndIconIII);
```

Comment:
```suggestion
				auto addr = get_pattern("c7 44 24 1c 00 00 00 00 c7 44 24 08 00 20 00", 0x4);
				
				HICON wndIconIII = LoadIcon(GetModuleHandle(nullptr), MAKEINTRESOURCE(0x412)); // Group icon ID
				Patch<HICON>(addr, wndIconIII);
```

Since you are writing a handle directly and not a pointer to it, loading the icon in advance is pointless. If you load it here, it also won't load needlessly in Vice City.

Reply:
That variable was needed there for the non-pattern version (to prevent duplication) and I rushed the pattern-based fix

Anyway that should be fixed now